### ISSUE TITLE
Xcode 9 , command : visualize error

### ISSUE THREAD

**Issue by HomlesZhao** [facebook/chisel#201]
look at 
```
error: error: cannot take the address of an rvalue of type 'CGFloat' (aka 'double')
Traceback (most recent call last):
  File "/usr/local/opt/chisel/libexec/fblldb.py", line 83, in runCommand
    command.run(args, options)
  File "/usr/local/Cellar/chisel/1.5.0/libexec/commands/FBVisualizationCommands.py", line 183, in run
    _visualize(arguments[0])
  File "/usr/local/Cellar/chisel/1.5.0/libexec/commands/FBVisualizationCommands.py", line 157, in _visualize
    _showLayer('[(id)' + target + ' layer]')
  File "/usr/local/Cellar/chisel/1.5.0/libexec/commands/FBVisualizationCommands.py", line 105, in _showLayer
    width = float(fb.evaluateExpression(size + '.width'))
TypeError: float() argument must be a string or a number
```

**Comment by kastiglione** [comment]
thanks for the report. If you want to try #202 and let me know, I'll merge it soon.

**Comment by bencochran** [comment]
I experienced the same issue, and the fix in #202 fixes it for me as well 👍
